### PR TITLE
WIP trusty cookbook updates

### PR DESCRIPTION
**Pull request by Lyoness**

### CODE REVIEW

**Comment by Lyoness** [comment]
testing out chris-lea redis-server ppa here 
https://github.com/travis-ci/travis-cookbooks/issues/836

File: cookbooks/travis_build_environment/recipes/redis.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
-apt_repository 'rwky-redis' do
-  uri 'http://ppa.launchpad.net/rwky/redis/ubuntu'
+apt_repository 'chris-lea-redis-server' do
+  uri 'http://ppa.launchpad.net/chris-lea/redis-server/ubuntu'
   distribution node['lsb']['codename']
   components ['main']
-  key '5862E31D'
-  keyserver 'hkp://ha.pool.sks-keyservers.net'
+  key 'C7917B12'
+  keyserver 'hkp://keyserver.ubuntu.com'
```
Comment (meatballhat) [inline cookbooks/travis_build_environment/recipes/redis.rb]: This key should be on `ha.pool.sks-keyservers.net`, too, which is generally much more reliable that the ubuntu keyserver.  Was it failing for you?

**Comment by meatballhat** [comment]
If this PR is targeting `master`, then it will only apply to Trusty stacks, IIUC 😄

**Comment by emdantrim** [comment]
@meatballhat @Lyoness: do we want to apply this to trusty only or also to precise? (also-to-precise makes sense to me.)

**Comment by emdantrim** [comment]
so we should make a separate PR for `precise-stable`, then?

File: cookbooks/travis_build_environment/recipes/redis.rb (hunk 1 above)
Comment (Lyoness) [inline cookbooks/travis_build_environment/recipes/redis.rb]: @meatballhat  no, rather that when I queried for the ppa and extract key on that keyserver, I couldn't find it (on the gui and cli)  Is there and I just didn't look right?

**Pull request closed without merging**